docs/fundamentals/client/client.md

```diff
@@ -668,7 +663,26 @@ with Flow().add() as f, open('output.txt', 'w') as fp:
     )
 ```
 
+````{admonition} What errors can be handled by the callback
+:class: caution
+Callback can handle errors that are caused by Executors raising an Exception.
+It will not receive exceptions or from the Gateway having connectivity errors with the Executors or between the Client and the Gateway.
```

Comment:
```suggestion
Callbacks can handle errors that are caused by Executors raising an Exception.

A callback will not receive exceptions:
- from the Gateway having connectivity errors with the Executors.
- between the Client and the Gateway.
```

I'm not clear on the "or from" part here. Is this a typo? Regardless, I've changed it as best I can. Does the admonition box support bullets?